src/whisk.rs

```diff
@@ -89,7 +95,7 @@ pub fn generate_whisk_shuffle_proof<T: RngCore>(
     rng: &mut T,
```

Comment:
Comment of this func should be updated as well I think, both to reflec that `m` is no longer returned and to also reflect that the shuffle proof is returned as bytes and not as a struct.